### ISSUE TITLE
Nether alignement

### ISSUE THREAD

**Issue by Toamastar** [Querz/mcaselector#85]
I have had much success exporting and importing overworld chunks, this program works great, thank you. I have been importing some other worlds into the empty regions of my own world and I am now trying to import the nether from those other worlds and want to aligne those chunks with their overworld ones so that the nether portals remain synced, The worlds I am importing have their own unique nether hubs which i would like to sync up and aligne correctly. Is there an easy way to figure this out with maths or can it be done within the program?

I have been wanting to do this for years, this program is the only way I have found to merge multiple worlds together, now all i want is for the nether hubs to work exactly as they did in their orignal worlds when I import the chunks. Apologies if this is hard to explain, thank you.

**Comment by Querz** [comment]
The distance between portals in the Nether is 1/8 of the distance of the portals in the Overworld, so you could calculate the location of the portals in the Nether by dividing the coordinates of the portal in the Overworld by 8 and then import those chunks with the appropriate offset.

Make sure to use the [latest release](https://github.com/Querz/mcaselector/releases/latest) to import chunks.

**Comment by Toamastar** [comment]
Thank you for the timely response! This seems like it may be difficult to be exact, I have been using offsets like 1000 x -500 to place them into the unloaded areas of my world. This seems like it may be difficult to figure out the offset in the nether due to the sheer size of the worlds im placing. Do i figure out the offset from the centre of the imported chunks? Do I divide the 1000 x -500 by 8 and it will be placed into the correct position in the nether? :)

I fear I may be overcomplicating things, i really appreciate the work you are doing, this program is amazing!!

**Comment by Querz** [comment]
What might work is that you write down all chunk locations of your Nether portals in the Overworld after importing all chunks of your Overworld into one world.
Divide all those coordinates by 8.
Get the chunk coordinates of all Nether portals from all worlds where you want to import them from.
Make a list of which Nether portal of your new Overworld should lead to which Nether portal later.
Calculate the differences between the two coordinate sets in your list and import the Nether chunks separately with those offsets.

This is just a theory though based on my current knowledge of how Nether portals work.

**Comment by Toamastar** [comment]
Thanks for the advice, this will be a lot of work, its hurting my head just thinking about it lol The more I think about it the more I understand it though. I will try a few things and get back to you. If I get it to work, I hope this can contribute to making this easier for others in the future :)

**Comment by Querz** [comment]
Did you get this to work?

**Comment by Toamastar** [comment]
Hi again!

I Didn't end up trying it yet, I got distracted with other things. I did some working out for a few days after our first conversation but couldn't figure out an easy and time effective way to align the chunks containing portals correctly with the rest of of the chunks. I don't understand the mechanics well enough to attempt it without working everything out first.

If I may give feedback, it would be helpful if we could visualise where the imported chunks were going to appear before actually importing them. A "drag and drop" feature with the mouse would also be helpful for operations like this. Importing chunks with the current interface feels a bit like a guessing game of where they will actually appear, this may be down to me misunderstanding how the program works fundamentally but I have gotten success with the overworld chunks so far.

If it could be done, Would like to see an outline of the chunks being imported and be able to drag it to where I want in the world or use the arrow keys to nudge it into place.

Thank you for your continued work on this project, its still really really cool! :)
